ldk/javascript/examples/self-test-loop/src/tests/network/index.ts

```diff
@@ -117,49 +116,47 @@ export const testWebsocketConnection = (): Promise<boolean> =>
     };
 
     try {
-      console.debug('Stating websocket connect');
       const socket = await network.webSocketConnect(socketConfiguration);
-      console.debug('Websocket successfully connected');
-      socket.setCloseHandler((error, code, text) => {
+      console.info('Websocket successfully connected');
+
+      socket.setCloseHandler(async (error, code, text) => {
         if (error) {
-          console.error(`OnCloseHandler received error:`);
           console.error(error);
-
-          return;
+          reject(error);
         }
-
         console.info(`Received on close code: ${code}. ${text}`);
+        if (textTestPassed && binaryTestPassed) {
```

Comment:
`textTestPassed` and `binaryTestPassed` are false here, may I know why we want it to resolve true with false `textTestPassed` and false `binaryTestPassed`

Reply:
Well, this is a callback scope. So the idea is to only resolve test as passed if we already received 2 messages: text and binary, before we get into this callback.